routes/subscriptions/SubscriptionController.cs

```diff
@@ -0,0 +1,90 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using TAIBackend.Model;
+
+namespace TAIBackend.routes.subscriptions;
+
+[Route("subscriptions")]
+public class SubscriptionController : Controller
+{
+    [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme)]
+    [HttpGet("me")]
+    public async Task<IActionResult> GetSubscibedAccounts(YoutubeContext db)
+    {
+        try
+        {
+            var ownerAccountId = Int64.Parse(User.FindFirst(ClaimTypes.NameIdentifier)!.Value);
+            var subs = await db.Subscriptions.Where(s => s.Owneraccountid == ownerAccountId).ToListAsync();
+            return Ok(new
+            {
+                data = subs.ToArray().Select(s => new
+                {
+                    id = s.Id, subscribedId = s.Owneraccountid
+                }),
+                count = subs.Count
+            });
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e);
+            throw;
+        }
+    }
+
+    [HttpPost("user/{userId}/subscribe")]
+    [Authorize(AuthenticationSchemes = CookieAuthenticationDefaults.AuthenticationScheme)]
+    [RequiresUserAccount]
+    public async Task<IActionResult> CreateSubscription(YoutubeContext db, long userId)
+    {
+        try
+        {
+            await db.Accounts.SingleAsync(a => a.Id == userId);
```

Comment:
niepotrzebna linijka. W POST nie potrzebujemy pobierać danych

Reply:
potrzebujemy, tylko trzeba sprawdzić czy entry jeszcze nie istnieje
oczywiście tego nie zrobilem xd

Reply:
no to ten problem by załatwiło gdyby kluczem głównym było złożenie dwóch kluczy obcych w tabeli subskrypcji

Reply:
ale najwyżej tak się to puści, żeby to było na pokazanie dzisiaj i się przerobi jak będzie sprint na spłacanie długu xdd

Reply:
sprint to może być do lidla